data_structures/tests/serializers.rs

```diff
@@ -305,11 +304,14 @@ fn message_version_from_bytes() {
             receiver_address: receiver_address,
             user_agent: "asdf".to_string(),
             last_epoch: 8,
-            genesis: 2,
             nonce: 1,
         }),
         magic: 1,
     };
+
+    // TODO: Ensure protobuf decryption, because this buf
+    // still have the genesis field of the old `Version` message
+    // and this test actually pass
```

Comment:
This is expected behavior: protocol buffers are backwards compatible, so unknown extra fields are ignored.

Reply:
Thanks, it was that!!